### PR TITLE
Render price calculator inline on large screens

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Render `PriceCalculator` inline on large screens instead of in modal.

## Justify why they are needed

Follow design.

I think we need to revise some layout later but let's merge this so we can show designers how it would work.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1069** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1069" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1069?utm_source=stack-comment).

**Review by alebedev — COMMENTED** [review]
I thought we agreed to deprecate our `useBreakpoint` and use CSS-based visibility instead?

File: apps/store/src/components/ProductPage/PurchaseForm/PurchaseForm.tsx [hunk 1]
```diff
@@ -130,6 +131,19 @@ type EditingStateProps = {
 const EditingState = (props: EditingStateProps) => {
   const { onToggleDialog, priceIntent, onSuccess } = props
   const { priceTemplate, story } = useProductPageContext()
+  const isLarge = useBreakpoint('lg')
+
+  const Children = (
```
Comment (alebedev) [inline apps/store/src/components/ProductPage/PurchaseForm/PurchaseForm.tsx:136]: Nit: `children`, it's not a class / component

**Review by alebedev — APPROVED** [review]
Looks good.  Changing `useBreakpoint` deserves another PR, I suspect

**Pull request merged**